Mathlib/Analysis/InnerProductSpace/PiL2.lean

```diff
@@ -612,6 +612,25 @@ protected theorem repr_reindex (b : OrthonormalBasis ι 𝕜 E) (e : ι ≃ ι')
 
 end OrthonormalBasis
 
+section EuclideanSpace
+
+variable (𝕜 ι) [IsROrC 𝕜] [Fintype ι] [DecidableEq ι]
+
+/-- The basis `Pi.basisFun` bundled as an orthormal basis of `EuclideanSpace 𝕜 ι`. -/
+noncomputable def EuclideanSpace.basisFun :
+    OrthonormalBasis ι 𝕜 (EuclideanSpace 𝕜 ι) := ⟨LinearIsometryEquiv.refl _ _⟩
```

Comment:
I think this might even be `default`?

Reply:
(in which case, `default` should be defined in terms of this)

Reply:
Indeed, this is the default OrthonormalBasis... I defined the default `OrthonormalBasis` using `EuclideanSpace.basisFun`. I hope that's what you had in mind...